app/models.py

```diff
@@ -1467,6 +1467,7 @@ def serialize(self):
         data.update({
             'id': self.id,
             'briefId': self.brief_id,
+            'briefTitle': self.brief.data.get('title', ''),
```

Comment:
It would feel bad to essentially start rewriting `brief.serialize()` whenever we want information about a brief on the brief response as it doesn't feel very DRY. We could add a key for `brief` and it's value would be the serialized brief using the already existing`brief.serialize()`... but then we would need to not include foreign properties as you could end up with an infinite loop (brief response contains brief which contains brief responses which contain the brief)...